pkg/gardenlet/controller/seed/seed_care_control.go

```diff
@@ -199,8 +199,16 @@ func (c *Controller) seedCareUpdate(_, newObj interface{}) {
 		return
 	}
 
-	if newSeed.Generation == newSeed.Status.ObservedGeneration && apiequality.Semantic.DeepEqual(oldSeed.Status, newSeed.Status) {
+	if apiequality.Semantic.DeepEqual(gardencorev1beta1helper.GetCondition(newSeed.Status.Conditions, gardencorev1beta1.SeedBootstrapped),
```

Comment:
This is not, what I meant with https://github.com/gardener/gardener/pull/5274#discussion_r858463933.

Isn't the controller trigger itself in an endless loop now?

I was suggesting to only enqueue seeds on update events if their `Bootstrapped` condition changed from something other than `True` to `True` just like the shoot care controller enqueues shoots on update events if their reconciliation just finished (by looking at the `lastOperation`).